.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+---
+
+**Description:**
+Describe your proposal.
+
+**Related issue:**
+Add issue link.
+
+**Pull request:**
+Add pull request link (if it exists).
```

Comment:
I am not sure that this field makes sense. Pull request should be provided only after issue creation and approval from task owner. It is better to mention issue in pull-request description

Reply:
Removed.